spec/views/hq/admin_users/new.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper' 
+
+describe 'hq/admin_users/new.html.erb', type: :view do
+
+  before :each do
+    assign :admin_user, build_stubbed(:admin_user)
+  end
+
+  it 'renders the form' do
+    render
+    expect(view).to render_template(:new)
+  end
+
+  it 'cancel button goes to index page' do
+    render
+    expect(rendered).to have_link('Cancel', href: hq_admin_users_path)
+  end
+
+end
```

Comment:
Should we also test that the view has a form, has all the 3 input fields and has a submit button?